providers/frame/zombienet.ts

```diff
@@ -57,7 +58,26 @@ export class ZombienetProvider extends FrameProxyProvider {
   zombienetMemo = new PermanentMemo<string, Network>()
   zombienet(configPath: string): Promise<Network> {
     return this.zombienetMemo.run(configPath, async () => {
-      const tmpDir = Deno.makeTempDirSync({ prefix: `capi_zombienet_` })
+      const tmpDir = Deno.realPathSync(Deno.makeTempDirSync({ prefix: `capi_zombienet_` }))
```

Comment:
Why are these sync?

Reply:
Doesn't make a difference here, does it? Either way, just made both async.

Reply:
Not a huge deal, but it does block the whole process on those operations when they're sync.